application/deed/utils.py

```diff
@@ -172,9 +204,27 @@ def convert_json_to_xml(deed_json):  # pragma: no cover
     deed_data_xml.set_Id("deedData")
     deed_data_xml.set_mdRef(deed_json["md_ref"])
     deed_data_xml.set_titleNumber(deed_json["title_number"])
-    deed_data_xml.set_propertyDescription("property description")
+    deed_data_xml.set_propertyDescription(deed_json["property_address"])
+
+    charge_clause_xml = api.chargeClauseType()
+    charge_clause_xml.set_creCode(deed_json["charge_clause"]["cre_code"])
+    charge_clause_xml.set_entryText(deed_json["charge_clause"]["description"])
+    deed_data_xml.set_chargeClause(charge_clause_xml)
+
+    additional_provisions = api.additionalProvisionsType()
+
+    for idx, provision_json in enumerate(deed_json["additional_provisions"]):
+        additional_provisions.add_provision(convert_json_to_provision(provision_json, idx))
+
+    deed_data_xml.set_additionalProvisions(additional_provisions)
+
+    deed_data_xml.set_lender(convert_json_to_lender(deed_json["lender"]))
+
+    deed_data_xml.set_effectiveClause("effective_clause")
+    # deed_data_xml.set_effectiveClause(deed_json["effective_clause"])
```

Comment:
Think we need to use the commented out value as the effective clause has now been implemented